### PR TITLE
Read default levels v2

### PR DESCRIPTION
**Pull request by dvcopae**

### CODE REVIEW

File: packages/instrumenter/src/mutation-level/default-mutation-levels.json
Comment (Ja4pp) [inline packages/instrumenter/src/mutation-level/default-mutation-levels.json:1]: Just as a remark! Based on the discussion we had in the chat! I might need small adaptations but that can be a separate issues and isn't required for this PR.

File: packages/instrumenter/src/mutation-level/mutation-level.ts
Comment (Ja4pp) [inline packages/instrumenter/src/mutation-level/mutation-level.ts:1]: ```../../../src/mutation-level/default-mutation-levels.json'``` 
will this work for all kind of projects, as it seems like it depends on the setup of someone?

Comment (dvcopae) [reply packages/instrumenter/src/mutation-level/mutation-level.ts:1]: In this case It doesn't matter where stryker is ran from. In this case `../../..` points to the `instrumenter` folder, as the file is normally ran from `dist/src/instrumenter/mutation-level`, where json files aren't generated.

Comment (Ja4pp) [reply packages/instrumenter/src/mutation-level/mutation-level.ts:1]: image


What am I doing wrong then? I made a console.log(defaultLevels) and imported the const

**Pull request merged**